markdown_it/main.py

```diff
@@ -27,36 +27,51 @@
     linkify_it = None
 
 
-_PRESETS = AttrDict(
-    {
-        "default": presets.default.make(),
-        "zero": presets.zero.make(),
-        "commonmark": presets.commonmark.make(),
-    }
-)
+_PRESETS = {
+    "default": presets.default.make(),
+    "zero": presets.zero.make(),
+    "commonmark": presets.commonmark.make(),
+}
+_PRESETS["js-default"] = _PRESETS["default"]
+_PRESETS["gfm-like"] = presets.default.make()
+_PRESETS["gfm-like"]["options"]["linkify"] = True
```

Comment:
Should we make these new presets public in `markdown_it.presets` package like the original ones are? That enables a workflow where you fetch a preset from there, mutate it, and pass it on to `.configure()`

Reply:
done in bc3a2efae6660a1a27ff28b540d6a7ee2bd7c430